packages/trace-viewer/index.html

```diff
@@ -26,5 +26,13 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.tsx"></script>
+    <dialog id="fallback-error">
+      <p>The Playwright Trace Viewer must be loaded over the <code>http://</code> or <code>https://</code> protocols.</p>
+      <p>For more information, please see the <a href="https://aka.ms/playwright/err-trace-viewer-file-protocol">docs</a>.</p>
```

Comment:
@mxschmitt what do you think about this link? In the interest of time, since we don't think folks will likely end up on this error anyway, we can also just omit the second paragraph for now and let folks file and issue or ask for help. (It at least won't be a blank screen any more and has minimal copy.)

Reply:
```suggestion
      <p>For more information, please see the <a href="https://aka.ms/playwright/trace-viewer-file-protocol">docs</a>.</p>
```
lgtm